igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/bitbucket/CommitController.groovy

```diff
@@ -68,11 +70,11 @@ class CommitController extends AbstractCommitController {
       }
     } catch (RetrofitError e) {
       if (e.getKind() == RetrofitError.Kind.NETWORK) {
-        throw new RuntimeException("Could not find the server ${bitBucketMaster.baseUrl}")
+        throw new NotFoundException("Could not find the server ${bitBucketMaster.baseUrl}")
```

Comment:
Is there a common cause for the `NETWORK` error? I wouldn't necessarily equate it to a 404 unless is a blatant misconfiguration or something and the config is pointed at an endpoint that doesn't respond?

Reply:
Yeah, I'm not really sure what's going on in this controller, it's always had this logic since the class was added. I'm going to go out on a limb and just delete that conditional.